setup/prepare-crc.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#This script configures CRC for running app-studio in it
+#It cofigures the CRC to have better memory and CPUs 
+#It also configures CRC to have required addons.
+
+#TODO: Check for CRC installation and binary location and set it to CRC-BINARY
+CRCBINARY=$(readlink -f ~/.crc/bin/crc)
```

Comment:
If I remember well, `readlink -f` does not work on macOS. @amfred could you check?

Reply:
That is correct.  On Macs, the default CRC installation also links the CRC command to `/usr/local/bin/crc`